rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -121,7 +121,7 @@ def init_check(self):
         # checks that messages are being receieved
         # (from both buggies if relevant)
         # covariance is less than 1 meter
-        if (self.self_odom_msg == None) or (self.has_other_buggy and self.other_odom_msg == None) or (self.self_odom_msg.pose.covariance[0] ** 2 + self.self_odom_msg.pose.covariance[7] ** 2 > 1**2):
+        if (self.self_odom_msg == None) or (self.self_odom_msg.pose.covariance[0] ** 2 + self.self_odom_msg.pose.covariance[7] ** 2 > 1**2):
```

Comment:
We should still print what is failing so that it is easier to debug

Reply:
added